### PR TITLE
Add option to not use minified javascript and ease development.

### PR DESCRIPTION
**Pull request by Carreau**
because I'm starting to be annoyed by the 30+s.

### CODE REVIEW

**Comment by minrk** [comment]
I'm fine with this, though I would reverse the logic, so the default behavior is to use the minified sources (e.g. if the variable is unspecified, the template will use main.min.js). I also think `use_bundles` is very hard to understand.

Perhaps `ignore_minified_js` better represents what's going on?

**Comment by Carreau** [comment]
Done

File: notebook/base/handlers.py [hunk 1]
```diff
@@ -117,7 +117,16 @@ class IPythonHandler(AuthenticatedHandler):
...
-    
+
+
+    @property
+    def ignore_minified_js(self):
+        """Wether to user bundle in template. (*.min files)
+        
+        Mainly use for development and avoid file recompilation
+        """
+        return self.settings.get('ignore_minified_js', True)
```
Comment (minrk) [inline notebook/base/handlers.py]: default should be False

**Comment by minrk** [comment]
:+1: other than one True that should be False

**Comment by Carreau** [comment]
Rhaa nice catch, I remembered to changed the default fro the traitlets, but not here. 

Done.

**Pull request merged**